homeassistant/components/stookwijzer/sensor.py

```diff
@@ -23,44 +65,40 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up Stookwijzer sensor from a config entry."""
-    client = hass.data[DOMAIN][entry.entry_id]
-    async_add_entities([StookwijzerSensor(client, entry)], update_before_add=True)
+
+    async_add_entities(
+        StookwijzerSensor(description, entry) for description in STOOKWIJZER_SENSORS
+    )
 
 
-class StookwijzerSensor(SensorEntity):
-    """Defines a Stookwijzer binary sensor."""
+class StookwijzerSensor(CoordinatorEntity, SensorEntity):
```

Comment:
```suggestion
class StookwijzerSensor(CoordinatorEntity[..], SensorEntity):
```
Please add the coordinator type here

Reply:
Yup, adjusted in my local version (otherwise mypy will not be happy :D )